From: Philip Withnall <withnall@endlessm.com>
Date: Tue, 11 Feb 2020 11:47:03 +0000
Subject: =?utf-8?q?=5BPATCH=5D_docs=3A_Don=E2=80=99t_install_object_manager?=
 =?utf-8?q?_example_separately?=

The relevant parts of the generated example documentation are already
`xi:include`d into the `migrating-gdbus.xml` page, so are turned into
HTML there. Installing them separately means they also get installed
into `/usr/share/gtk-doc/html/gdbus-object-manager-example/`, which
seems redundant.

Signed-off-by: Philip Withnall <withnall@endlessm.com>

Bug-Upstream: https://gitlab.gnome.org/GNOME/glib/merge_requests/1359
---
 docs/reference/gio/gdbus-object-manager-example/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/docs/reference/gio/gdbus-object-manager-example/meson.build b/docs/reference/gio/gdbus-object-manager-example/meson.build
index 126e677..13cae5b 100644
--- a/docs/reference/gio/gdbus-object-manager-example/meson.build
+++ b/docs/reference/gio/gdbus-object-manager-example/meson.build
@@ -6,5 +6,6 @@ gdbus_object_manager_example_doc = gnome.gtkdoc('gdbus-object-manager-example',
   scan_args : gtkdoc_common_scan_args + [
     '--rebuild-types',
   ],
+  install : false,
 )
 
